### ISSUE TITLE
jonn4y i need your help

### ISSUE THREAD

**Issue by cryptonana** [3s3s/opentrade#176]
jonn4y i need your help about chat messenger, i wana add time date in chat box, and tell me you sync all coins blockchain full node or use lite nodes

**Comment by cryptonana** [comment]
and dear one more thing how you done vol BTC when i set its show me Vol: NaN BTC this please help

**Comment by CryptorClub** [comment]
You need to write his nickname with @ symbol @jonn4y , we will be happy, for your attention

**Comment by IMPERIUM-main-dev** [comment]
@CryptorClub 

You worked with jonn4y  ?????

**Comment by CryptorClub** [comment]
@IMPERIUM-main-dev no. You may see, that we use opentrade exchange on different projects.

**Comment by knkrth** [comment]
@jonn4y As you know there is no new updates from @3s3s Can you share your(@jonn4y) updated source code with us?
